WalletWasabi.Fluent/Models/Wallets/CoinModel.cs

```diff
@@ -10,17 +11,36 @@ namespace WalletWasabi.Fluent.Models.Wallets;
 [AutoInterface]
 public partial class CoinModel : ReactiveObject
 {
+	[AutoNotify] private bool _isExcludedFromCoinJoin;
+	[AutoNotify] private int _anonymitySet;
+	[AutoNotify] private bool _confirmed;
+	[AutoNotify] private bool _isCoinJoinInProgress;
+	[AutoNotify] private bool _isBanned;
+	[AutoNotify] private string? _bannedUntilUtcToolTip;
+	[AutoNotify] private string? _confirmedToolTip;
+
 	public CoinModel(Wallet wallet, SmartCoin coin)
 	{
+		Coin = coin;
 		PrivacyLevel = coin.GetPrivacyLevel(wallet.AnonScoreTarget);
 		Amount = coin.Amount;
 		IsConfirmed = coin.Confirmed;
 		Confirmations = coin.GetConfirmations();
 		AnonScore = (int)coin.AnonymitySet;
 		Labels = coin.GetLabels(wallet.AnonScoreTarget);
 		Key = coin.Outpoint.GetHashCode();
+
+		this.WhenAnyValue(c => c.Coin.IsExcludedFromCoinJoin).BindTo(this, x => x.IsExcludedFromCoinJoin);
+		this.WhenAnyValue(c => c.Coin.Confirmed).BindTo(this, x => x.Confirmed);
+		this.WhenAnyValue(c => c.Coin.HdPubKey.AnonymitySet).Subscribe(x => AnonymitySet = (int)x);
```

Comment:
The Model's `AnonScore` is the value of `AnonimitySet`. So this line should store the value in `AnonScore`.
Also if that happens, the L29 feels unnecessary.
You could also use `BindTo` paired with `Select` for consistency